workflow-job

```diff
@@ -296,13 +302,17 @@ flows["${sha1}"] = {
     statuses.addPendingStatus(dockeringSpringJPA);
     def dockeringGuiceCassandraLdap = commitStatusFactory.from("Building Guice Cassandra/Ldap");
     statuses.addPendingStatus(dockeringGuiceCassandraLdap);
+    def dockeringGuiceJPASampleTesting = commitStatusFactory.from("Dockering Guice JPA with data provision");
+    statuses.addPendingStatus(dockeringGuiceJPASampleTesting);
 
     def deployingGuiceCassandra = commitStatusFactory.from("Deploying Guice Cassandra");
     statuses.addPendingStatus(deployingGuiceCassandra);
     def deployingGuiceJPA = commitStatusFactory.from("Deploying Guice JPA");
     statuses.addPendingStatus(deployingGuiceJPA);
     def deployingSpringJPA = commitStatusFactory.from("Deploying Spring JPA");
     statuses.addPendingStatus(deployingSpringJPA);
+    def deployingGuiceJPASampleTesting = commitStatusFactory.from("Deploying Guice JPA with data provision");
```

Comment:
What's repeated? One is dockering and the other is deploying. Those are not the same